library/ssl_tls13_generic.c

```diff
@@ -1505,4 +1505,42 @@ int mbedtls_ssl_tls13_generate_and_write_ecdh_key_exchange(
 }
 #endif /* MBEDTLS_ECDH_C */
 
+#if defined(MBEDTLS_KEY_EXCHANGE_SOME_PSK_ENABLED)
+/* Check if we have any PSK to offer, returns 0 if PSK is available.
+ * Assign the psk and ticket if pointers are present.
+ */
+MBEDTLS_CHECK_RETURN_CRITICAL
+int mbedtls_ssl_get_psk_to_offer(
```

Comment:
It is not a `static int` function
```suggestion
int mbedtls_ssl_get_psk_to_offer(
```

Reply:
Seems it's defined as "__warn_unused_result__", what's the relationship with `static int`?  Actually, do we have the standard that which return value should be checked unused or not.

Reply:
For `int` functions exported by a C file, the way it has been done so far is to add `MBEDTLS_CHECK_RETURN_CRITICAL` in the header file (ssl_misc.h for that function) that exposes the function but not in the C file. Thus yes we do not need it here.

Reply:
Not addressed yet.